contrib/mesos/pkg/executor/executor.go

```diff
@@ -415,9 +415,18 @@ func (k *Executor) handleChangedApiserverPod(pod *api.Pod) {
 	}
 
 	oldPod := k.pods[task.podName]
+	if oldPod == nil {
+		// TODO(jdef) should sent a TASK_FAILED or TASK_LOST here
```

Comment:
possibly racy: the pod could have been scheduled (launched) and then labels updated. before we process the LaunchTask in this executor we could receive the labels update via the apiserver watch. so we wouldn't find the pod here even though it's totally valid -- we just haven't received/processed the LaunchTask that would allow us to recognize it